QA.md

```diff
@@ -49,6 +49,14 @@
 - [ ] It's possible to go to the next screen by pressing <kbd>enter</kbd> if
       the input field validations allow it
 - [ ] Handle and passphrase validations work
+  - handle
+    - starts with a letter or digit
+    - is at least 2 characters long
+    - can contain the letters `a`-`z`, `A`-`Z`, digits `0`-`9` and special
+      characters `-`, `_`
```

Comment:
While positive cases are good, I think negative cases should be here too. Even just examples the person could copy and paste into the field.

Reply:
In my experience the best bugs pop up when people try to use software without any instructions, that's why I thought it might be valuable to let it open to interpretation.

Reply:
But can also add a list of positive and negative examples if you still think it's valuable.

Reply:
```
- [ ] Go wild!
```

:smile: I agree but we should also have consistent examples that we expect to fail/pass for quality assurance purposes :)

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/941/commits/e8e7097d8ad3e93bc99ce0b1df955427b736ff12

Reply:
And maybe for projects as well? :grimacing:

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/941/commits/b3669a283c3dc657f73a96920d626b7cfb9ed484